kola/tests/bpf/local-gadget.go

```diff
@@ -0,0 +1,160 @@
+// Copyright © Microsoft Corporation
```

Comment:
I would prefer to have a neutral term like `Copyright 2021 The mantle Authors` in https://github.com/flatcar-linux/mantle/blob/0282d298e43b0a99e17faaf9a0396d10a6fab4e0/kola/tests/util/template.go#L1 .

Reply:
Added the following headers:
```
// Copyright The Mantle Authors.
// SPDX-License-Identifier: Apache-2.0
```

Reply:
Thanks for updating the headers. Looks good.